### ISSUE TITLE
War1gus rescue wounded issue.

### ISSUE THREAD

**Issue by Zombie-Ryushu** [Wargus/war1gus#181]
I am unable to rescue the wounded troops in the Human campaign. When healed, they remain the immovable Wounded sprite.

**Comment by Zombie-Ryushu** [comment]
Update: This only happened because I reloaded from a saved game. 
When ever I reload from a save, all unit colors are black, mine and that of the enemies. 
Upon starting the mission fresh without reloading a save, I could see the proper colors and heal the units correctly.

**Comment by timfel** [comment]
Yes, there are multiple campaign missions that do not support save and load, they have to be played from start. It's on the list :)
